packages/x-charts-pro/package.json

```diff
@@ -40,7 +40,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.25.0",
-    "@mui/system": "^5.16.7",
+    "@mui/system": "^5.16.6 || ^6.0.0-rc.0",
```

Comment:
Huh, how does this work?

Reply:
Wait, I don't think it's gonna work.
The reason it works in our pipeline is that the `use-material-ui-v6` script overwrites the `@mui/system` dependency version in all MUI X packages:

<img width="360" src="https://github.com/user-attachments/assets/969f1f6c-bd79-4e29-91a9-d470e451b1e3">

This is not what will be published to the NPM registry though.

Reply:
Thanks for clarifying.
I was just testing an idea we discussed on `code-infra` meeting that @Janpot suggested.
Done some preliminary testing and it seems to behave strangely...
It looks like install with `npm` resolves `@mui/system` correctly—to v5 when `@mui/material` is v5 and to v6 when `@mui/material@6` is used.
But pnpm downloads both v5 and v6 of `@mui/system`. 🙈